.gitignore

```diff
@@ -1,2 +1,7 @@
 node_modules
 jspm_packages
+
+# Ignore built files
+systemjs-babel-node.js
+regenerator-runtime.js
+systemjs-babel-browser.js
```

Comment:
This is correct but I'm still not sure if I want to remove the build files just yet. I still like being able to download them from github...